cmd/pkger/cmds/pack.go

```diff
@@ -124,20 +121,23 @@ func (e *packCmd) Flags() *flag.FlagSet {
 }
 
 func Package(info here.Info, out string, decls parser.Decls) error {
-	os.RemoveAll(out)
```

Comment:
You want to keep this to make sure the original `pkged.go` file, if there is one, is gone before you doing any work.

Reply:
Yes, I kept it, it became `os.RemoveAll(fp)` still line 127

Reply:
I removed the same line in `Exec()`